msteams-platform/apps-in-teams-meetings/teams-live-share-canvas.md

```diff
@@ -0,0 +1,313 @@
+| [InkingManager](/javascript/api/@microsoft/live-share-canvas/inkingmanager) | Class that attaches a `<canvas>` element to a given `<div>` to automatically manage pen/highlighter strokes, laser pointer, lines & arrows, erasers, and cursors. Exposes a set of APIs to control which tool is active and basic configuration settings. |
+| [LiveCanvas](/javascript/api/@microsoft/live-share-canvas/livecanvas)       | A `SharedObject` class that synchronizes strokes & cursor positions from `InkingManager` for everyone in a Live Share session.                                                                                                                            |
+
+Example:
+
+```html
+<body>
+  <div id="canvas-host"></div>
+</body>
+```
+
+# [JavaScript](#tab/javascript)
+
+```javascript
+import * as microsoftTeams from "@microsoft/teams-js";
+import { TeamsFluidClient } from "@microsoft/live-share";
+import { InkingManager, LiveCanvas } from "@microsoft/live-share-canvas";
+
+// Initialize the Teams Client SDK
+await microsoftTeams.app.initialize();
+
+// Setup the Fluid container
+const client = new TeamsFluidClient();
+const schema = {
+  initialObjects: { liveCanvas: LiveCanvas },
+};
+const { container } = await client.joinContainer(schema);
+const { liveCanvas } = container.initialObjects;
+
+// Get the canvas host element
+const canvasHostElement = document.getElementById("canvas-host");
+const inkingManager = new InkingManager(canvasHostElement);
+
+// Begin synchronization for LiveCanvas
+await liveCanvas.initialize(inkingManager);
+```
+
+# [TypeScript](#tab/typescript)
+
+```TypeScript
+import * as microsoftTeams from "@microsoft/teams-js";
+import { TeamsFluidClient } from "@microsoft/live-share";
+import { InkingManager, LiveCanvas } from "@microsoft/live-share-canvas";
+import { ContainerSchema } from "fluid-framework";
+
+// Initialize the Teams Client SDK
+await microsoftTeams.app.initialize();
+
+// Setup the Fluid container
+const client = new TeamsFluidClient();
+const schema: ContainerSchema = {
+  initialObjects: { liveCanvas: LiveCanvas },
+};
+const { container } = await client.joinContainer(schema);
+const liveCanvas = container.initialObjects.liveCanvas as LiveCanvas;
+
+// Get the canvas host element
+const canvasHostElement = document.getElementById("canvas-host");
+const inkingManager = new InkingManager(canvasHostElement);
+
+// Begin synchronization for LiveCanvas
+await liveCanvas.initialize(inkingManager);
+```
+
+---
+
+## Canvas tools & cursors
+
+Now that you understand how to set up the Live Share canvas and it is being synchronized, you can now configure the canvas for user interaction, such as with buttons to select a pen tool.
```

Comment:
Maybe remove "now" in "you can now configure" since the whole sentence starts with "Now"